lib/tests/streamlit/connections/__init__.py

```diff
@@ -11,6 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-# Explicitly re-export public symbols.
-from streamlit.connections.base_connection import BaseConnection as BaseConnection
```

Comment:
I'm not sure how this ended up being added here -- must have just done so accidentally